### ISSUE TITLE
eman2.tests.test_protocols_eman.TestEmanTomoSubtomogramRefinement fails

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-eman2#82]
```
00016:   STARTED: refinementSubtomogram, step 2, time 2020-07-24 11:03:12.781144
00017:   /lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/bin/e2spt_refine.py   Runs/000172_EmanProtTomoRefinement/extra/subtomograms/overview_wbp-2.hdf --reference=Runs/000109_EmanProtTomoExtraction/extra/overview_wbp-1.mrc  --mass=500.000000 --goldstandard=-1  --pkeep=0.800000  --sym=c1  --maxtilt=90.0  --path=Runs/000172_EmanProtTomoRefinement/extra/spt_00  --niter=2 --parallel=thread:4 --threads=4
00018:   e2proclst.py Runs/000172_EmanProtTomoRefinement/extra/subtomograms/overview_wbp-2.hdf --create Runs/000172_EmanProtTomoRefinement/extra/spt_00/input_ptcls.lst
00019:   Traceback (most recent call last):
00020:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/bin/e2proclst.py", line 365, in <module>
00021:       main()
00022:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/bin/e2proclst.py", line 178, in main
00023:       n=EMUtil.get_image_count(f)
00024:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/lib/python2.7/site-packages/EMAN2db.py", line 545, in db_get_image_count
00025:       raise Exception(fsp)
00026:   Exception: Runs/000172_EmanProtTomoRefinement/extra/subtomograms/overview_wbp-2.hdf
00027:   Traceback (most recent call last):
00028:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/bin/e2spt_refine.py", line 236, in <module>
00029:       main()
00030:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/bin/e2spt_refine.py", line 108, in main
00031:       ep=EMData(ptcls,0)
00032:     File "/lmb/home/gsharov/soft/scipion3/software/em/eman-2.31/lib/python2.7/site-packages/EMAN2db.py", line 386, in db_emd_init
00033:       self.__initc(*parms)
00034:   RuntimeError: OutofRangeException at /home/eman2/anaconda2/conda-bld/eman2_1569858012581/work/libEM/imageio.cpp:101: '0 out of range [0,-1]' caught
```

**Comment by pconesa** [comment]
This might have been fixed wit #83

**Comment by azazellochg** [comment]
yes it was
